posthog/tasks/exports/test/test_csv_exporter_renders.py

```diff
@@ -0,0 +1,49 @@
+import json
+import os
+from unittest.mock import Mock, patch
+
+import pytest
+
+from posthog.models import ExportedAsset
+from posthog.tasks.exports import csv_exporter
+from posthog.test.base import APIBaseTest
+
+TEST_BUCKET = "Test-Exports"
+
+directory = os.path.join(os.path.abspath(os.path.dirname(__file__)), "./csv_renders")
+fixtures = []
+
+for file in os.listdir(directory):
+    filename = os.fsdecode(file)
+    if filename.endswith(".json"):
+        fixtures.append(filename)
+
+
+class TestCSVExporterRenders(APIBaseTest):
+    @patch("posthog.tasks.exports.csv_exporter.requests.request")
+    def test_response_renders(self, mock_request) -> None:
+        with self.settings(OBJECT_STORAGE_ENABLED=False):
+            for filename in fixtures:
```

Comment:
love this approach

Would be cool if it generated multiple tests instead of multiple assertions....